pkg/destinations/redshift/insert.go

```diff
@@ -0,0 +1,149 @@
+package redshift
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"os"
+	"strings"
+
+	"github.com/rs/zerolog/log"
+	"github.com/scratchdata/scratchdata/util"
+
+	"github.com/scratchdata/scratchdata/pkg/storage/blobstore/s3"
+)
+
+func (s *RedshiftServer) createColumns(table string, jsonTypes map[string]string) error {
+
+	for colName, jsonType := range jsonTypes {
+		colType := "VARCHAR"
+		switch jsonType {
+		case "int":
+			colType = "BIGINT"
+		case "bool":
+			colType = "BOOLEAN"
+		case "float":
+			colType = "DOUBLE PRECISION"
+		case "string":
+			colType = "VARCHAR"
+		default:
+			colType = "VARCHAR"
+		}
+
+		sql := fmt.Sprintf("ALTER TABLE \"%s\" ADD COLUMN \"%s\" %s", table, colName, colType)
+		_, err := s.conn.Exec(sql)
+		if err != nil {
+			if !strings.Contains(err.Error(), "already exists") {
+				log.Err(err).Msg("createColumns: cannot create column")
```

Comment:
```suggestion
				log.Error().Err(err).Str("column", colName).Msg("createColumns: cannot create column")
```

Reply:
While we don't need `Error().Err()`, we still want to avoid `Msgf()` in favor of `Str("column", colName)`. This will let us index logs easier.

Reply:
Noted